crates/biome_grit_patterns/src/pattern_compiler.rs

```diff
@@ -237,262 +226,3 @@ impl PatternCompiler {
         }
     }
 }
-
-impl PatternCompiler {
-    pub(crate) fn from_snippet_node(
```

Comment:
This method has been refactored and moved to `snippet_compiler.rs`, since that's where it's used anyway.